### PR TITLE
Remove build history and deployment history from main tables

### PR DESCRIPTION
**Pull request by jwforres**
Also:
Fixes https://github.com/openshift/origin/issues/5055

### CODE REVIEW

**Comment by jwforres** [comment]
@spadgett review even though i know I will probably have to rebase later :)

**Comment by jwforres** [comment]
cc @sg00dwin

File: assets/app/views/builds.html [hunk 1]
```diff
@@ -36,7 +35,8 @@
               <td>&nbsp;</td>
             </tr>
             <!-- Build config with builds, or builds whose build config has since been deleted -->
-            <tr ng-repeat="build in buildConfigBuilds | orderObjectsByDate : true" ng-if="$first || expandedBuildConfigRow[buildConfigName]">
+            <!-- We only show the first build from a build config for now-->
+            <tr ng-repeat="build in buildConfigBuilds | orderObjectsByDate : true" ng-if="$first">
```
Comment (spadgett) [inline assets/app/views/builds.html]: Instead of `ng-if`, how about `| limitTo: 1`?

File: assets/app/views/builds.html [hunk 2]
```diff
@@ -36,7 +35,8 @@
...
               <td>
                 <span ng-if="$first">
```
Comment (spadgett) [inline assets/app/views/builds.html]: Is `<span ng-if="$first">` still needed?

File: assets/app/views/deployments.html [hunk 3]
```diff
@@ -34,7 +33,7 @@
               <td>&nbsp;</td>
             </tr>
             <!-- Deployment config with deployments, or deployments from a deployment config which has since been deleted -->
-            <tr ng-repeat="deployment in deploymentConfigDeployments | orderObjectsByDate : true" ng-if="deploymentConfigName != '' && ($first || expandedDeploymentConfigRow[deploymentConfigName])">
+            <tr ng-repeat="deployment in deploymentConfigDeployments | orderObjectsByDate : true" ng-if="deploymentConfigName != '' && $first">
               <td>
                 <span ng-if="$first || deploymentConfigName == ''">
```
Comment (spadgett) [inline assets/app/views/deployments.html]: Still needed?

File: assets/app/views/deployments.html [hunk 4]
```diff
@@ -34,7 +33,7 @@
               <td>&nbsp;</td>
             </tr>
             <!-- Deployment config with deployments, or deployments from a deployment config which has since been deleted -->
-            <tr ng-repeat="deployment in deploymentConfigDeployments | orderObjectsByDate : true" ng-if="deploymentConfigName != '' && ($first || expandedDeploymentConfigRow[deploymentConfigName])">
+            <tr ng-repeat="deployment in deploymentConfigDeployments | orderObjectsByDate : true" ng-if="deploymentConfigName != '' && $first">
```
Comment (spadgett) [inline assets/app/views/deployments.html]: I think `limitTo: 1` will work here as well.

**Comment by jwforres** [comment]
[merge]

@sg00dwin fyi my branch's commit history changed due to rebasing so you'll need to reset your mobile branch on master once this merges and cherrypick your changes.  You might also hit a merge conflict in the templates due to the changes that sam requested.

**Comment by jwforres** [comment]
unrelated failure  https://gist.github.com/jwforres/6821da6aa3dd4a2b0ada

[merge]

**Comment by jwforres** [comment]
and again [merge]

**Pull request merged**